adapter/src/model/ProtocolConfig.ts

```diff
@@ -0,0 +1,35 @@
+import { Protocol } from "./enum/Protocol";
+import { validators } from '@jvalue/node-dry-basics';
+
+export interface ProtocolConfig {
+
+    protocol:Protocol;
+    parameters: Map<string, any> | undefined;
```

Comment:
I'd probably make it a `Record<string, unknown>`. Maybe makes sense to model the concrete values in the inheriting interfaces explicitly..